apps/fz_http/test/fz_http_web/live/setting_live/client_defaults_test.exs

```diff
@@ -92,6 +92,32 @@ defmodule FzHttpWeb.SettingLive.ClientDefaultsTest do
              """
     end
 
+    test "blocks overridden default client endpoint" do
+      FzHttp.Config.put_system_env_override(:default_client_endpoint, "1.2.3.4:1234")
+
+      {_admin_user, conn} = admin_conn(%{})
```

Comment:
We need live view to start after the config is overridden, otherwise, the page is already rendered when we try to set config update and there is no helper function in LiveView to re-render it (call `update/2` again).

Reply:
This makes sense.